### PR TITLE
Added data-types validation.

### PR DESCRIPTION
**Pull request by kadosh**
This is a partial fix for #114

### CODE REVIEW

File: dist/tubular-bundle.es5.js [hunk 1]
```diff
@@ -4206,15 +4209,15 @@
             $ctrl.dataType = columns[0].DataType;
             $ctrl.filterOperators = filterOperators[$ctrl.dataType];
 
-            if ($ctrl.dataType === 'date' || $ctrl.dataType === 'datetime' || $ctrl.dataType === 'datetimeutc') {
+            if ($ctrl.dataType === dataTypes.DATE || $ctrl.dataType === dataTypes.DATE_TIME || $ctrl.dataType === dataTypes.DATE_TIME_UTC) {
                 $ctrl.filter.Argument = [new Date()];
 
                 if ($ctrl.filter.Operator === 'Contains') {
```
Comment (Alexey-Unosquare) [inline dist/tubular-bundle.es5.js:4215]: Why not use data type constants here?

File: src/js/tubular/models/tubular-column.js [hunk 2]
```diff
@@ -10,16 +10,21 @@
          * @description
          * The `tubularColumn` factory is the base to generate a column model to use with `tbGrid`.
          */
-        .factory('tubularColumn', [function() {
-            return function(columnName, options) {
+        .factory('tubularColumn', ['dataTypes', function (dataTypes) {
+            return function (columnName, options) {
                 options = options || {};
-                
+                options.DataType = options.DataType || 'string';
```
Comment (Alexey-Unosquare) [inline src/js/tubular/models/tubular-column.js:16]: dataTypes.STRING maybe?

File: dist/tubular-bundle.es5.js (hunk 1 above)
Comment (kadosh) [reply dist/tubular-bundle.es5.js:4215]: Fixed on latest commits

File: src/js/tubular/models/tubular-column.js (hunk 2 above)
Comment (kadosh) [reply src/js/tubular/models/tubular-column.js:16]: Fixed on latest commits

Comment (geoperez) [reply src/js/tubular/models/tubular-column.js:16]: You need to sync your branch with master please

**Pull request merged**